Diverse small fixes

levitte opened the pull request:
- Replace `$title$` with `$breadcrumb$` in `inc/pandoc-body-epilogue.html5`,
  'cause that's actually how it is used.  They avoids interference from
  pandoc, which has ideas of its own what `$title$` should be used for.
- Alter the CSS for tables to work with different ways of wring them.
  That makes these three selectors equivalent:

      table > tr:first-child
      table > thead
      table > :not(thead + tbody) > tr:first-child

  That also makes these three selectors equivalent:

      table > tr:nth-child(even)
      table > thead + tbody > tr:nth-child(odd)
      table > :not(thead + tbody) > tr:nth-child(even)

levitte committed:
Diverse small fixes

- Replace `$title$` with `$breadcrumb$` in `inc/pandoc-body-epilogue.html5`,
  'cause that's actually how it is used.  They avoids interference from
  pandoc, which has ideas of its own what `$title$` should be used for.

levitte commented:
This turned out to work the way we want.  See ongoing test on https://beta.openssl.org

levitte commented:
Note that this is required for all of #343, #344, #345, #346, #347, #348 and #349 to render properly

levitte commented:
Approval needed from another @openssl/omc

levitte committed:
Diverse small fixes

- Alter the CSS for tables to work with different ways of wring them.
  That makes these two selectors equivalent:

      table > tr:first-child
      table > thead

  That also makes these two selectors equivalent:

      table > tr:nth-child(even)
      table > tbody > tr:nth-child(odd)

levitte commented:
I added a CSS mod, necessary for more properly (from an HTML5 sense) formatted tables :wink:

t8m commented:
> I added a CSS mod, necessary for more properly (from an HTML5 sense) formatted tables wink

You'll also need to update the HTML based tables to add the thead?

t8m commented:
For example the news table on the beta.openssl.org/index.html looks now wrong.

levitte commented:
> For example the news table on the beta.openssl.org/index.html looks now wrong.

Argh...  while *we* don't in our HTML, modern browser automatically wrap all the \<tr>'s in a table with a \<tbody>...\</tbody>

levitte committed:
squash! Diverse small fixes

The selectors are now:

    table > tr:first-child
    table > thead
    table > :not(thead + tbody) > tr:first-child

and:

    table > tr:nth-child(even)
    table > thead + tbody > tr:nth-child(odd)
    table > :not(thead + tbody) > tr:nth-child(even)

levitte commented:
Ok, I amended the selectors to take into account \<tbody> without a preceding \<thead>

levitte commented:
@t8m, if you'd double-check the tables on beta.openssl.org again?

t8m commented:
> @t8m, if you'd double-check the tables on beta.openssl.org again?

They look fine now.

levitte commented:
> Approval needed from another @openssl/omc

Again

levitte commented:
PING

levitte commented:
Merged

017f25cfc4b4232ffca1e108919bf29dd25b92b2 Diverse small fixes
8b2adfd8fa74db148760da299d1a754bc0cf4fbf Diverse small fixes
